sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1580,22 +1580,37 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the length of a given string value
-   *
+   * Computes the length of a given string value.
+   * 
```

Comment:
A few benefits:

1. We don't need to depend on commons-util. In general it is good to reduce external dependency due to dependency conflicts.

2. Our version ideally should work directly against UTF8String, so we don't need conversion.